Orange/widgets/data/owfile.py

```diff
@@ -409,14 +409,12 @@ def _update_sheet_combo(self):
         self.sheet_box.show()
 
     def _select_active_sheet(self):
-        if self.reader.sheet:
-            try:
-                idx = self.reader.sheets.index(self.reader.sheet)
-                self.sheet_combo.setCurrentIndex(idx)
-            except ValueError:
-                # Requested sheet does not exist in this file
-                self.reader.select_sheet(None)
-        else:
+        try:
+            idx = self.reader.sheets.index(self.reader.sheet)
+            self.sheet_combo.setCurrentIndex(idx)
+        except ValueError:
+            # Requested sheet does not exist in this file
+            self.reader.select_sheet(None)
```

Comment:
Your PR used `select_sheet(0)` here. Was that intentional? I added a commit that changes it back to `None` (which is the current value for unknown/default sheet).

Reply:
@janezd

Reply:
(I thought I answered this, but now I can't find the answer. Perhaps I forgot to submit it.)

I guess this was needed at some iteration, but now seems to works without. You can merge with your change.

Problems on macOS are unrelated and also appeared in other PRs. I'll investigate.